drivers/SmartThings/zigbee-switch/fingerprints.yml

```diff
@@ -1,5 +1,35 @@
 zigbeeManufacturer:
   #HANSSEM SMART SWITCH
+  - id: "SMARTvill/SLA06"
+    deviceLabel: "SMARTvill SLA06"
+    manufacturer: SMARTvill
+    model: SLA06
+    deviceProfileName: switch-no-master-6
+  - id: "SMARTvill/SLA05"
+    deviceLabel: "SMARTvill SLA05"
+    manufacturer: SMARTvill
+    model: SLA05
+    deviceProfileName: switch-no-master-5
+  - id: "SMARTvill/SLA04"
+    deviceLabel: "SMARTvill SLA04"
+    manufacturer: SMARTvill
+    model: SLA04
+    deviceProfileName: switch-no-master-4
+  - id: "SMARTvill/SLA03"
+    deviceLabel: "SMARTvill SLA03"
+    manufacturer: SMARTvill
+    model: SLA03
+    deviceProfileName: switch-no-master-3
```

Comment:
Since this is a "multi-switch" device the profile assignment for these devices should all be to "basic-switch". Please change the profiles for these devices and test the changes. You should see a device card for each "component" on the device.